services/ui-src/src/utils/other/time.ts

```diff
@@ -13,12 +13,14 @@ export const noon: TimeShape = {
   second: 0,
 };
 
-export const calculateTimeByDateType = (dateType: string): TimeShape => {
+export const calculateTimeByType = (
+  timeType: string | undefined
+): TimeShape => {
   const timeMap: any = {
-    "abf-startDate": midnight,
-    "abf-endDate": oneSecondToMidnight,
+    startDate: midnight,
+    endDate: oneSecondToMidnight,
   };
-  return timeMap[dateType] || noon;
+  return timeMap?.[timeType as keyof typeof timeMap] || noon;
 };
```

Comment:
Updated to be field-name agnostic and instead rely on matching the passed 'timeType'.